sdv/lite/single_table.py

```diff
@@ -84,7 +92,7 @@ def sample(self, num_rows, max_tries_per_batch=100, batch_size=None,
                 The file to periodically write sampled rows to. If None, does not
                 write rows anywhere.
             conditions:
```

Comment:
Since we will be releasing a new major version I don't think we need to support this deprecated argument anymore